samples/sensor/vl53l0x/src/main.c

```diff
@@ -7,12 +7,13 @@
 #include <zephyr.h>
 #include <device.h>
 #include <sensor.h>
+#include <stdio.h>
```

Comment:
This change should be in a separate patch, it has nothing to do with sensor units.